docs/upctl_ip-address_modify.md

```diff
@@ -0,0 +1,31 @@
+## upctl ip-address modify
+
+Modify an IP address
+
+```
+upctl ip-address modify <Address/PTRRecord...> [flags]
+```
+
+### Options
+
+```
+      --mac string          MAC address of server interface to attach floating IP to.
+      --ptr-record string   A fully qualified domain name.
```

Comment:
probably should make it explicit that this flag is the *new* ptr record for the ip-address? it's not immediately clear that this is the update and not just a way of referring to it.